tests/einstein/operations/unit/test_operations.py

```diff
@@ -1,40 +1,48 @@
+# Tests the calls in einstein/operations.py
+
 import time
 import unittest
 from unittest import TestCase
 
+import ims.common.config as config
 import ims.common.constants as constants
 import ims.einstein.ceph as ceph
+from ims.common.log import trace
+from ims.database.database import Database
 from ims.einstein.operations import BMI
 
 _cfg = config.get()
 
```

Comment:
So that it is not used outside this module, python doesnt use enforce it, but it is a programmer thing. Any class, variable, method, etc with _ is not supposed to be used outside the module or class.